lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-vmware-infra_manager-host_esx.rb

```diff
@@ -1,119 +1,49 @@
 module MiqAeMethodService
   class MiqAeServiceManageIQ_Providers_Vmware_InfraManager_HostEsx < MiqAeServiceManageIQ_Providers_Vmware_InfraManager_Host
+    include MiqAeServiceEmsOperationsMixin
+    require_relative "mixins/miq_ae_service_retirement_mixin"
     def shutdown(force = false)
-      ar_method do
-        MiqQueue.put(
-          :class_name  => @object.class.name,
-          :instance_id => @object.id,
-          :method_name => "vim_shutdown",
-          :zone        => @object.my_zone,
-          :role        => "ems_operations",
-          :args        => [force]
-        ) if @object.is_vmware?
-        true
-      end
+      sync_or_async_ems_operation(false, "vim_shutdown", [force]) if @object.is_vmware?
```

Comment:
This is already a VMware model so `if @object.is_vmware?` can be dropped from all these methods.